applications/nublado/values-base.yaml

```diff
@@ -0,0 +1,78 @@
+controller:
+  config:
+    images:
+      source:
+        type: "docker"
+        registry: "ts-dockerhub.lsst.org"
+        repository: "sal-sciplat-lab"
+      num_releases: 0
+      num_weeklies: 3
+      num_dailies: 2
+      cycle: 31
+      recommended_tag: "recommended_c0031"
+    lab:
+      env:
+        DAF_BUTLER_REPOSITORY_INDEX: "/project/data-repos.yaml"
+        LSST_DDS_INTERFACE: "net1"
+        LSST_DDS_PARTITION_PREFIX: "base"
+        LSST_SITE: "base"
+        PGPASSFILE: "/opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt"
+        PGUSER: "oods"
+      volumes:
+      - containerPath: "/home"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/jhome"
+          server: "nfs-jhome.ls.lsst.org"
+      - containerPath: "/project"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/project"
+          server: "nfs-project.ls.lsst.org"
+      - containerPath: "/scratch"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/scratch"
+          server: "nfs-scratch.ls.lsst.org"
+      - containerPath: "/datasets"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/lsstdata"
+          server: "nfs-lsstdata.ls.lsst.org"
+      - containerPath: "/repo/LATISS"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/auxtel/repo/LATISS"
+          server: "nfs-auxtel.ls.lsst.org"
+      - containerPath: "/net/obs-env"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/obs-env"
+          server: "nfs-obsenv.ls.lsst.org"
+      - containerPath: "/data/lsstdata/BTS/auxtel"
+        source:
+          type: "nfs"
+          path: "/auxtel/lsstdata/BTS/auxtel"
+          server: "nfs-auxtel.ls.lsst.org"
+
+jupyterhub:
+  cull:
+    users: false
+    removeNamedServers: false
+    timeout: 432000
+    every: 300
+    maxAge: 2160000
+  singleuser:
+    extraAnnotations:
+      k8s.v1.cni.cncf.io/networks: "kube-system/dds"
+  hub:
+    baseUrl: "/n3"
+    db:
+      upgrade: true
+      url: "postgresql://jovyan@postgresdb01.ls.lsst.org/jupyterhub"
```

Comment:
This, and likewise elsewhere, is the same URL as is used by nublado2, but you're installing this in parallel at `/n3`. I don't think this can possibly work; if both hubs are running at the same time, they need to have separate databases.

Reply:
RIght you are.